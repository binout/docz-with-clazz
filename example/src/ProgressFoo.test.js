import React from "react";
import renderer from "react-test-renderer";

import ProgressFoo from "./ProgressFoo";

describe("<ProgressFoo />", () => {
  
  it("should match snapshot", () => {
    const cmp = renderer.create(
    // tag::basic[]
    <ProgressFoo now="50" />
    // end::basic[]
    ).toJSON();
    expect(cmp).toMatchSnapshot();
  });

});
