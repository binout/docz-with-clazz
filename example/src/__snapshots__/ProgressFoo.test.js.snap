// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressFoo /> should match snapshot 1`] = `
<div>
  50
</div>
`;
